CareKit/PDF/OCKDocument.m

```diff
@@ -174,7 +174,7 @@ - (instancetype)copyWithZone:(NSZone *)zone {
 }
 
 static NSString *imageTagFromView (UIView *view) {
-    UIGraphicsBeginImageContextWithOptions(view.frame.size, YES, 2.0);
+    UIGraphicsBeginImageContextWithOptions(view.frame.size, YES, 0.0);
```

Comment:
This means to output on a PDF file, should it be depends on the device's screen resolution?